online-shop/src/app/auth/services/auth.service.ts

```diff
@@ -0,0 +1,43 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { httpOptions } from 'src/app/shared/constants';
+import { LoginCredentials } from 'src/app/shared/models/LoginCredentials.model';
+import { User } from '../models/User.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  private loginUrl = "http://localhost:3000/login"
+  isLoggedIn = false;
+  user: User | null = null;
+  redirectUrl: string = "/products";
+
+  constructor(private http: HttpClient, private router: Router) { }
+
+  login(loginCredentials: LoginCredentials): Observable<User> {
+    console.log("entered login", loginCredentials);
```

Comment:
Never let console.log in the code. It should be used only for testing/debugging purposes
*check all places (files)